commit 6b850bca1d0558e5fdfcec42197777910b0bc8f0
Author: Timothy Potter <thelabdude@apache.org>
Date:   Thu Jan 14 05:51:17 2016 +0000

    SOLR-8456: Raise the time we wait to see replicas recover to allow for slower test envs like Jenkins
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1724537 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java b/solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java
index db5a26e..e36f5ce 100644
--- a/solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java
+++ b/solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java
@@ -74,7 +74,8 @@ public class HttpPartitionTest extends AbstractFullDistribZkTestBase {
   // recognizes (and propagates) partitions
   protected static final long sleepMsBeforeHealPartition = 2000L;
 
-  protected static final int maxWaitSecsToSeeAllActive = 30;
+  // give plenty of time for replicas to recover when running in slow Jenkins test envs
+  protected static final int maxWaitSecsToSeeAllActive = 90;
 
   public HttpPartitionTest() {
     super();
